openapi/components/schemas/CurrentUserPresence.yaml

```diff
@@ -4,6 +4,11 @@ properties:
   avatarThumbnail:
     type: string
     nullable: true
+  currentAvatarTags:
+    type: string
+  debugflag:
+    type: string
+    nullable: true
```

Comment:
```suggestion
```

debugflag are getting added and removed on all endpoints. we shouldn't document them

Reply:
I counter that with the fact that some of the SDKs, like the Java SDK in particular, throw an exception when an undocumented property is encountered, so the deserialization process itself fails, making the SDK completely unusable and unreliable.
If there's some option in generators that can fix that, that would be great, but otherwise I'd prefer to mark it as deprecated, if anything.

Reply:
> If there's some option in generators that can fix that, that would be great, but otherwise I'd prefer to mark it as deprecated, if anything.

There has to be a generator option for this

Reply:
Constantly updating debug flags sounds like something that is unmaintainable

Reply:
For the Java SDK in particular, I have figured out how to edit the schema classes' bytecode on-the-fly to partially alleviate this problem, but this is obviously something that should be usable out-of-the-box. Tangentially, I will probably at some point be publishing a Maven project with a bunch of utilities that the SDK doesn't have by default, including some WebSocket stuff. (Deserializing the WebSocket messages has caused me the most annoyance, but is also how I've noticed a bunch of the changes for which I've made PRs.)